web_sdk/common_test_utils/pubspec.yaml

```diff
@@ -0,0 +1,7 @@
+name: common_test_utils
```

Comment:
I'd name it `web_test_utils` because it's not common to the whole repo, but only for web.

Reply:
done.